conans/client/cmake.py

```diff
@@ -52,10 +53,11 @@ def _generator(self):
                 return base
 
         if operating_system == "Windows":
-            if compiler == "gcc":
-                return "MinGW Makefiles"
-            if compiler in ["clang", "apple-clang"]:
-                return "MinGW Makefiles"
+            non_msvc = (compiler != "Visual Studio")
```

Comment:
what about the case when we compile on Windows and we use Visual Studio then? If I turn this check off, then cmake will attempt to use MinGW Makefiles, or am I wrong?